atom/common/api/atom_bindings.cc

```diff
@@ -35,6 +41,14 @@ void FatalErrorCallback(const char* location, const char* message) {
   AtomBindings::Crash();
 }
 
+content::RenderFrame* GetCurrentRenderFrame() {
```

Comment:
Isn't it already implemented somewhere?

Reply:
yes. it was already duplicated in `remote_object_freer.cc`. I've refactored it to keep a single copy in `atom_api_renderer_ipc.h`